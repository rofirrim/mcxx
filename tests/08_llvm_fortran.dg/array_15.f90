! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="array_15.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    LOGICAL :: A(2), B(2), C(2)

    A(1) = .TRUE.
    A(2) = .FALSE.

    B(1) = .FALSE.
    B(2) = .TRUE.

    PRINT *, A
    PRINT *, B

    C = .NOT. A

    PRINT *, C
END PROGRAM MAIN
