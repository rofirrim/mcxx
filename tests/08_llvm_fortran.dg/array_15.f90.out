 T F
 F T
 F T
